repose-aggregator/tests/performance-tests/test_vars/services/uriredaction.yml

```diff
@@ -0,0 +1,29 @@
+{
+  dir_prefix: "services/uriredaction",
+  repose: {
+    config: {
+      system_model: {
+        services: [
+          # todo: when it becomes a service, add "uri-redaction"
+        ]
+      }
+    },
+    config_files: {
```

Comment:
I haven't looked through the Ansible tasks in a while, but will Repose run without specifying the system model and container configuration templates here? I seem to remember us being explicit when applying configuration.

Reply:
After looking through and confirming with @Mario-Lopez , the Repose perf tests are set up to have these [defaults](https://github.com/rackerlabs/repose/blob/master/repose-aggregator/tests/performance-tests/roles/repose/defaults/main.yml#L53-L57) as long as there are no specific `templates` defined.

Reply:
Ah, I see. Terrific!

Reply:
Man, I posted that comment a moment too soon.

I was just running the performance test to verify it, and it failed with the following:
```
failed: [perf-services-uriredaction-repose-test01] (item=system-model.cfg.xml) => {
    "changed": false,
    "item": "system-model.cfg.xml",
    "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'filters'"
}
```

So you are right that the system model and container configs will be defaulted, but templating of the system model is currently broken if no filters are defined.

I suggest adding a default for filters the way that we have a default for services here:
https://github.com/rackerlabs/repose/blob/master/repose-aggregator/tests/performance-tests/roles/repose/defaults/main.yml#L20

Reply:
Empty list of `filters` added to the default system model.